php.ini-development

```diff
@@ -772,54 +775,54 @@ user_dir =
 ; properly in multithreaded servers, such as IIS or Zeus, and is automatically
 ; disabled on them.
 ; https://php.net/enable-dl
 enable_dl = Off
 
 ; cgi.force_redirect is necessary to provide security running PHP as a CGI under
-; most web servers.  Left undefined, PHP turns this on by default.  You can
+; most web servers. Left undefined, PHP turns this on by default. You can
 ; turn it off here AT YOUR OWN RISK
 ; **You CAN safely turn this off for IIS, in fact, you MUST.**
 ; https://php.net/cgi.force-redirect
-;cgi.force_redirect = 1
+;cgi.force_redirect = On
 
 ; if cgi.nph is enabled it will force cgi to always sent Status: 200 with
 ; every request. PHP's default behavior is to disable this feature.
-;cgi.nph = 1
+;cgi.nph = On
 
 ; if cgi.force_redirect is turned on, and you are not running under Apache or Netscape
 ; (iPlanet) web servers, you MAY need to set an environment variable name that PHP
-; will look for to know it is OK to continue execution.  Setting this variable MAY
+; will look for to know it is OK to continue execution. Setting this variable MAY
 ; cause security issues, KNOW WHAT YOU ARE DOING FIRST.
 ; https://php.net/cgi.redirect-status-env
-;cgi.redirect_status_env =
+;cgi.redirect_status_env = ""
 
-; cgi.fix_pathinfo provides *real* PATH_INFO/PATH_TRANSLATED support for CGI.  PHP's
+; cgi.fix_pathinfo provides *real* PATH_INFO/PATH_TRANSLATED support for CGI. PHP's
 ; previous behaviour was to set PATH_TRANSLATED to SCRIPT_FILENAME, and to not grok
-; what PATH_INFO is.  For more information on PATH_INFO, see the cgi specs.  Setting
-; this to 1 will cause PHP CGI to fix its paths to conform to the spec.  A setting
-; of zero causes PHP to behave as before.  Default is 1.  You should fix your scripts
+; what PATH_INFO is. For more information on PATH_INFO, see the cgi specs. Setting
+; this to 1 will cause PHP CGI to fix its paths to conform to the spec. A setting
+; of zero causes PHP to behave as before. Default is 1. You should fix your scripts
 ; to use SCRIPT_FILENAME rather than PATH_TRANSLATED.
 ; https://php.net/cgi.fix-pathinfo
-;cgi.fix_pathinfo=1
+;cgi.fix_pathinfo = On
 
 ; if cgi.discard_path is enabled, the PHP CGI binary can safely be placed outside
 ; of the web tree and people will not be able to circumvent .htaccess security.
-;cgi.discard_path=1
+;cgi.discard_path = On
 
 ; FastCGI under IIS supports the ability to impersonate
-; security tokens of the calling client.  This allows IIS to define the
-; security context that the request runs under.  mod_fastcgi under Apache
+; security tokens of the calling client. This allows IIS to define the
+; security context that the request runs under. mod_fastcgi under Apache
 ; does not currently support this feature (03/17/2002)
-; Set to 1 if running under IIS.  Default is zero.
+; Set to 1 if running under IIS. Default is zero.
 ; https://php.net/fastcgi.impersonate
-;fastcgi.impersonate = 1
+;fastcgi.impersonate = "1"
```

Comment:
Is this a boolean INI setting or not? If not it _may_ make sense to convert it to one.

Reply:
According to the [documentation](https://www.php.net/manual/en/ini.core.php#ini.fastcgi.impersonate) this is string. Once it becomes bool I will fix this in the future PR.

Reply:
The documentation, especially for INI settings, is *not* the source of truth. The source code is.

Reply:
OK, thanks for the input. I'm gonna look up all of the directives inside the source code.

Reply:
https://github.com/php/php-src/blob/f90323c8d4456ff2b9a0f442f6a383d9b58185ae/sapi/cgi/cgi_main.c#L1516